oap-server/server-library/library-client/src/main/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ElasticSearchClient.java

```diff
@@ -300,7 +303,28 @@ public boolean existDoc(String indexName, String id) {
         return es.get().documents().exists(indexName, TYPE, id);
     }
 
-    public SearchResponse ids(String indexName, Iterable<String> ids) {
+
+    /**
+     * @since 9.2.0 Provide to get documents from multi indices by ids.
+     * @param indexIds key: indexName, value: ids list
+     * @return Documents
```

Comment:
```suggestion
     * Provide to get documents from multi indices by IDs.
     * @param indexIds key: indexName, value: ids list
     * @return Documents
     * @since 9.2.0
```